File: src/C/1009.c
/*
 * 1009 - Salary with Bonus
 * Author: Otavio
 * Date: 26/04/2024
 */
#include <stdio.h>

int
main()
{
	char name[100];
	double salary, profit;

	scanf("%s", name);
	scanf("%lf", &salary);
	scanf("%lf", &profit);

	printf("TOTAL = R$ %.2lf\n", salary+(profit*.15));
	return 0;
}

File: src/JS/1009.js
/*
 * 1009 - Salary with Bonus
 * Author: Otavio
 * Date: 31/08/2024
 */
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

lines.shift(); //discard the seller's name as it won't be used
const fixedSalary = parseFloat(lines.shift());
const totalSales = parseFloat(lines.shift());
const totalSalary = fixedSalary + (totalSales * 0.15);

console.log(`TOTAL = R$ ${totalSalary.toFixed(2)}`);
